src/Containers/SessionChoice/index.tsx

```diff
@@ -0,0 +1,87 @@
+import { Col, Row } from 'antd';
+import React, { useState } from 'react';
+
+import Button from 'Components/Button';
+import Input from 'Components/Input';
+import { Box, CenteredContainer, OptionalText, Tags } from 'Components/CommonComponents';
+import pushNotification from 'pushNotification';
+
+interface Payload {
+  nickname: string;
+  game_code: string;
+}
+
+interface Response {
+  player_id: number;
+  game_code: number;
+  token: string;
+}
+
+const SessionChoice = () => {
+  const [nickname, setNickname] = useState<string>('');
+  const [gameCode, setGameCode] = useState<string>('');
+  const [loading, setLoading] = useState<boolean>(false);
+
+  const newSessionButton = 'Create new session';
+  const joinSessionButton = 'Join session';
+
+  const onSubmit = async () => {
+    const info: Payload = { nickname, game_code: gameCode };
+    setLoading(true);
+    try {
+      const response = await fetch('http://127.0.0.1:8000/api/v1/game/', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(info),
+      });
+      if (response.ok) {
+        const result: Response = await response.json();
+        localStorage.setItem('token', result.token);
+        localStorage.setItem('player_id', String(result.player_id));
+        localStorage.setItem('game_code', String(result.game_code));
+        pushNotification('success', 'Joining server', 'Enjoy the game');
+      } else {
+        const { errors } = await response.json();
+        Object.entries(errors).forEach(([key, value]) => {
+          pushNotification('warning', `${key}: ${(value as string[]).join(' and ')}`);
+        });
+      }
```

Comment:
```suggestion
    const { token, player_id, game_code, errors } = await response.json();
      if (response.ok) {
        localStorage.setItem('token', token);
        localStorage.setItem('player_id', String(player_id));
        localStorage.setItem('game_code', String(game_code));
        pushNotification('success', 'Joining server', 'Enjoy the game');
      } else {
        Object.entries(errors).forEach(([key, value]) => {
          pushNotification('warning', `${key}: ${(value as string[]).join(' and ')}`);
        });
      }
```

Reply:
Remove interface too

Reply:
I think its good practice to make interface to data from fetch, beacuse we can inform typescript about type of data, but okay, its new language for me i will change it 😢